scalameta/testkit/src/main/scala/scala/meta/testkit/Corpus.scala

```diff
@@ -0,0 +1,57 @@
+package scala.meta.testkit
+
+/** A collection of Scala source files to run [[SyntaxAnalysis]].
+  *
+  * @param url A zip file that matches the structure of this directory:
+  *            https://github.com/olafurpg/scala-repos/tree/master/repos
```

Comment:
How do we make sure that this link remains active throughout the history of scala.meta?